core/commands/bitswap.go

```diff
@@ -64,18 +65,24 @@ Print out all blocks currently on the bitswap wantlist for the local peer.`,
 			if err != nil {
 				return err
 			}
-			if pid == nd.Identity {
-				return cmds.EmitOnce(res, &KeyList{bs.GetWantlist()})
+			if pid != nd.Identity {
+				return cmds.EmitOnce(res, &KeyList{bs.WantlistForPeer(pid)})
 			}
-
-			return cmds.EmitOnce(res, &KeyList{bs.WantlistForPeer(pid)})
 		}
+
 		return cmds.EmitOnce(res, &KeyList{bs.GetWantlist()})
 	},
 	Encoders: cmds.EncoderMap{
 		cmds.Text: cmds.MakeTypedEncoder(func(req *cmds.Request, w io.Writer, out *KeyList) error {
+			// sort the keys first
+			keys := make([]string, 0, len(out.Keys))
```

Comment:
So, we can actually just:

1. import `github.com/ipfs/go-cidutil`.
2. Call `cidutil.Sort(out.Keys)`

Reply:
@Stebalien Thx your advise. I had changed it.